phpstan.neon.dist

```diff
@@ -97,6 +97,10 @@ parameters:
             message: '~^Parameter #2 \$lockMode of method Doctrine\\DBAL\\Platforms\\AbstractPlatform\:\:appendLockHint\(\) expects 0\|1\|2\|4, 128 given\.$~'
             path: tests/Platforms/AbstractPlatformTestCase.php
 
+        # Type check for legacy implementations of the Connection interface
+        # TODO: remove in 4.0.0
+        - "~Call to function method_exists\\(\\) with Doctrine\\\\DBAL\\\\Driver\\\\Connection and 'getNativeConnection' will always evaluate to true\\.~"
```

Comment:
This patch adds an error suppression but doesn't seem to be adding the code that causes the error. If this line is removed, the build still passes.